src/Exceptions/InvalidCommandException.php

```diff
@@ -32,5 +37,41 @@ public function render(Console $console): void
         if ($missingArguments) {
             $console->writeln("Missing arguments: {$missingArguments}");
         }
+
+        $fixedArguments = $this->promptForMissingArguments($console);
+
+        foreach ($fixedArguments as $argument) {
+            $this->bag->add($argument);
+        }
+
+        throw new MistypedCommandException($this->consoleCommand->getName());
+    }
+
+    /**
+     * @return array
+     * @throws Exception
+     */
+    private function promptForMissingArguments(Console $console): array
+    {
+        $arguments = [];
+
+        foreach ($this->invalidDefinitions as $definition) {
+            if (! $definition->hasDefault) {
+                $value = $this->resolveValue($definition, $console);
+
+                $arguments[] = new ConsoleInputArgument(name: $definition->name, position: $definition->position, value: $value);
+            }
+        }
+
+        return $arguments;
+    }
+
+    private function resolveValue(ConsoleArgumentDefinition $definition, Console $console): mixed
```

Comment:
We could expand this so "help" for specific definition would also get written to the screen